### ISSUE TITLE
Vertical Displays

### ISSUE THREAD

**Issue by esetnik** [JohnCoates/Aerial#264]
It would be nice to have the option to use scale to fit rather than scale to fill for vertical displays. The screen savers look quite pixelated when stretched vertically by 2x or more.

Thanks for the hard work!

**Comment by ccharlton** [comment]
I also use portrait displays, so anything around better support for them is always a gift.

**Comment by juandesant** [comment]
I also use vertical displays, but find that scale to fill helps with bringing the focus. The screensaver activates when I'm away, so I like to find the screen full with the video, not so much the resolution. 

Also, by getting the videos from Apple, we can get better versions if they ever provide better versions —4K content, for instance, for a next-generation AppleTV—, and I'm wary of using different sources.

Long story short, if you implement this, please make it a preference.

**Comment by glouel** [comment]
Closing as fixed with the release of 1.4.1
